app/lib/Plugins/SearchEngine/Elastic8/FieldTypes/Intrinsic.php

```diff
@@ -152,21 +152,45 @@ public function getIndexingFragment($content, array $options): array {
 		}
 
 		if ($rel_type_id = caGetOption('relationship_type_id', $options)) {
-			$return[caGetRelationshipTypeCode($rel_type_id) . $this->getDataTypeSuffix(self::SUFFIX_KEYWORD)]
-				= $content;
+			$return[caGetRelationshipTypeCode($rel_type_id) ?: $rel_type_id] = $return;
```

Comment:
Yeah, I have some specific questions on the ticket itself, but I wanted to add to the current data rather than completely replacing it. 

Original:
```
    "ca_occurrences/occurrence_id": [
      {
        "-i": 1,
      }
    ],
 ```
 New:
 ```   
        "ca_occurrences/occurrence_id": [
      {
        "-i": 1,
        "contains_is_part_of_object": {
          "-i": 1
        }
      }
    ],
 ```
 
 Happy to yank this commit pending further conversation?